corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -36,7 +38,7 @@ hqDefine("data_dictionary/js/data_dictionary", [
         return self;
     };
 
-    var propertyListItem = function (name, isGroup, groupName, caseType, dataType, description, deprecated) {
+    var propertyListItem = function (name, isGroup, groupName, caseType, dataType, description, fhirRsourcePropPath, deprecated) {
```

Comment:
Optional: I'm a fan of passing `options` object when param lists get long and then using [assert_properties](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/assert_properties.js) to verify expected params are included. It also lets you distinguish between required and optional, which would be nice here where the fhir properties are their own flag.

Reply:
That makes sense. Going to skip this for now @orangejenny . 
I tried that i would just need to do something like
```
assertProperties.assert(details, [], ['id', 'build_id', 'active', 'app', 'version', 'location',
                'activated_on', 'deactivated_on']);
```
but this would need more headspace to understand the place this is being called from too.